adding search attributes for python (#1360)
temporalio/documentation: docs/python/how-to-upsert-custom-search-attributes-to-a-workflow-executions-during-execution-in-python.md

---
id: how-to-upsert-custom-search-attributes-to-a-workflow-executions-during-execution-in-python

> title: How to upsert custom search attributes
> sidebar_label: Upsert custom search attributes
> description: Upsert custom search attributes

Reviewer: ```suggestion
title: How to upsert custom Search Attributes
sidebar_label: Upsert custom Search Attributes
description: To upsert custom Search Attributes, use [`upsert_search_attributes()`] and set it to an empty list.
```

tags:
  - developer-guide
  - sdk
  - python
---

> To upsert custom search attributes, use the [`upsert_search_attributes()`](https://python.temporal.io/temporalio.workflow.html#upsert_search_attributes) function and set it to an empty list.
> 
> The keys will be added or replaced on top of the existing search attributes, similar to [`dict.update()`](https://docs.python.org/3/library/stdtypes.html#dict.update).

Reviewer: ```suggestion
To upsert custom Search Attributes, use the [`upsert_search_attributes()`](https://python.temporal.io/temporalio.workflow.html#upsert_search_attributes) function and set it to an empty list.

The keys are added to or replace the existing Search Attributes, similar to [`dict.update()`](https://docs.python.org/3/library/stdtypes.html#dict.update).
```
